api/PerformanceNavigation.json

```
@@ -29,10 +29,10 @@
             "version_added": "14"
           },
           "safari": {
-            "version_added": false
+            "version_added": "11"
```

```suggestion
            "version_added": "8"
```

I assume this change was needed because the lint would otherwise fail, but it looks like `performance.navigation` and all members except `toJSON` were supported since Safari 8, confirmed by testing 7.1 and 8 on https://mdn-bcd-collector.appspot.com/tests/api/PerformanceNavigation.

This matches the data in https://developer.mozilla.org/en-US/docs/Web/API/Performance/navigation#browser_compatibility.

It's not necessary for this PR, but it would be nice to also update the `redirectCount` and `type` entries to "8".